docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -690,6 +861,411 @@ Your Devfile has been successfully deployed
+      apiVersion: apps/v1
+      metadata:
+        name: {{RESOURCE_NAME}}
+      spec:
+        replicas: 1
+        selector:
+          matchLabels:
+            app: {{RESOURCE_NAME}}
+        template:
+          metadata:
+            labels:
+              app: {{RESOURCE_NAME}}
+          spec:
+            containers:
+              - name: {{RESOURCE_NAME}}
+                image: {{CONTAINER_IMAGE}}
+                ports:
+                  - name: http
+                    containerPort: {{CONTAINER_PORT}}
+                    protocol: TCP
+                resources:
+                  limits:
+                    memory: "1024Mi"
+                    cpu: "500m"
+
+# This will create a Service so your Deployment is accessible.
+# Depending on your cluster, you may modify this code so it's a
+# NodePort, ClusterIP or a LoadBalancer service.
+- name: outerloop-service
+  kubernetes:
+    inlined: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: {{RESOURCE_NAME}}
+      spec:
+        ports:
+        - name: "{{CONTAINER_PORT}}"
+          port: {{CONTAINER_PORT}}
+          protocol: TCP
+          targetPort: {{CONTAINER_PORT}}
+        selector:
+          app: {{RESOURCE_NAME}}
+        type: ClusterIP
+
+# Let's create an Ingress so we can access the application via a domain name
+- name: outerloop-ingress
+  kubernetes:
+    inlined: |
+      apiVersion: networking.k8s.io/v1
+      kind: Ingress
+      metadata:
+        name: {{RESOURCE_NAME}}
+      spec:
+        rules:
+          - host: "{{DOMAIN_NAME}}"
+            http:
+              paths:
+                - path: "/"
+                  pathType: Prefix
+                  backend:
+                    service:
+                      name: {{RESOURCE_NAME}} 
+                      port:
+                        number: {{CONTAINER_PORT}}
+```
+
+
+#### 3. Run the `odo deploycommand
+
+Now we're ready to run `odo deploy`:
+
+```console
+$ odo deploy
+  __
+ /  \__     Deploying the application using my-go-app Devfile
+ \__/  \    Namespace: default
+ /  \__/    odo version: v3.0.0-alpha2
+ \__/
+
+↪ Building & Pushing Container: MYUSERNAME/go-example
+ •  Building image locally  ...
+ ✓  Building image locally [880ms]
+ •  Pushing image to container registry  ...
+ ✓  Pushing image to container registry [5s]
+
+↪ Deploying Kubernetes Component: go-example
+ ✓  Searching resource in cluster
+ ✓  Creating kind Deployment [48ms]
+
+↪ Deploying Kubernetes Component: go-example
+ ✓  Searching resource in cluster
+ ✓  Creating kind Service [51ms]
+
+↪ Deploying Kubernetes Component: go-example
+ ✓  Searching resource in cluster
+ ✓  Creating kind Ingress [49ms]
+
+Your Devfile has been successfully deployed
+```
+
+Your application has now been deployed to the Kubernetes cluster with Deployment, Service, and Ingress resources.
+
 Test your application by visiting the `DOMAIN_NAME` variable that you had set in the `devfile.yaml`.
 
```

Comment:
This quickstart guide does not cover the cleanup after `odo deploy`, do you think it would help to have that?